app/controllers/projects_controller.rb

```diff
@@ -53,12 +54,20 @@ def destroy
 
   private
 
+  def create_project
+    @create_project ||=
+      CreateProject.call(project_params: project_params, current_user: current_user)
+  end
+
+  def update_project
+    UpdateProject.call(project_params: project_params, current_user: current_user, project_id: @project.id.to_s)
```

Comment:
Why do you pass project_id instead of just project?

Reply:
Fixed